From ddd657bfa68412e9a25f028cb44b69197a39f409 Mon Sep 17 00:00:00 2001
Message-Id: <ddd657bfa68412e9a25f028cb44b69197a39f409.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Tue, 22 Jul 2014 21:27:03 +0530
Subject: [PATCH 203/312] MUST_REBASE [VPG]: drm/i915: Enable DPST on CHV

Add DPST support for chv by enabling has_dpst in cherryview intel_device_info

!FOR_UPSTREAM: DPST feature is not to be upstreamed, as the corresponding
User mode component can not be open sourced.

Change-Id: I178ba766aceba96e864bf35d763185c4027076e4
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_drv.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i915/i915_drv.c b/drivers/gpu/drm/i915/i915_drv.c
index ad52f7e..7497ddd 100644
--- a/drivers/gpu/drm/i915/i915_drv.c
+++ b/drivers/gpu/drm/i915/i915_drv.c
@@ -349,6 +349,7 @@ static const struct intel_device_info intel_cherryview_info = {
 	.need_gfx_hws = 1, .has_hotplug = 1,
 	.ring_mask = RENDER_RING | BSD_RING | BLT_RING | VEBOX_RING,
 	.is_valleyview = 1,
+	.has_dpst = 1,
 	.display_mmio_offset = VLV_DISPLAY_BASE,
 	GEN_CHV_PIPEOFFSETS,
 	CURSOR_OFFSETS,
-- 
1.7.9.5

